packages/flame_flare/example/lib/main.dart

```diff
@@ -19,8 +19,6 @@ class MyGame extends FlameGame with TapDetector, DoubleTapDetector {
 
   late final MinionComponent minionComponent;
 
-  bool loaded = false;
```

Comment:
Note: this wasn't being used and was conflicting with the new getter, so I just removed it